packages/api-client/rollup.config.js

```diff
@@ -75,10 +75,10 @@ const umd = defineConfig({
     input: 'src/index.ts',
     output: {
         dir: 'dist',
-        entryFileNames: 'umd/[name].umd.js',
+        entryFileNames: '[name].umd.js',
         format: 'umd',
         sourcemap: true,
-        name: 'opendatasoft.apiclient',
+        name: 'opendatasoft.visualizations',
```

Comment:
I think somehow your PR reverted another one of my PR: https://github.com/opendatasoft/ods-dataviz-sdk/pull/110

Maybe something with all the rebase funk? 😅 But if it's unvoluntary, I think it's probably easier to just re-commit the 2 lines that changed, and not jump back into Git problems...

Reply:
No I might have really CC there. I'll double check the output directories too.